package.nls.json

```diff
@@ -36,6 +40,8 @@
     "AWS.lambda.policy.unknown.function": "Unknown",
     "AWS.message.credentials.error": "There was an issue trying to use credentials profile {0}.\nYou will be disconnected from AWS.\n\n{1}",
     "AWS.message.enterProfileName": "Enter the name of the credential profile to use",
+    "AWS.message.info.cloudFormation.delete": "{0} was deleted. Please check for any retained resources at AWS Console",
+    "AWS.message.error.cloudFormation.delete": "An error occured while deleting {0}. Please check the stack events at AWS Console",
```

Comment:
Please change to
`An error occurred while deleting CloudFormation Stack {0}. Please check the stack events on the AWS Console.`